app/views/query/index.html.erb

```diff
@@ -22,7 +22,9 @@
 <% if @results %>
   <div class="container">
     <div class="pb-2">
-      <h1>SQL Query Results</h1>
+      <h1>SQL Query Results &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp; 
```

Comment:
use bootstrap instead of multiple spaces

Reply:
I used the bootstrap float utility class and removed the HTML non breaking space tabs.